bundle/openapi.go

```diff
@@ -15,68 +15,14 @@
 package bundle
 
 import (
-	"bytes"
-	"encoding/json"
-	"errors"
-	"fmt"
-
 	"github.com/erda-project/erda/apistructs"
-	"github.com/erda-project/erda/bundle/apierrors"
-	"github.com/erda-project/erda/pkg/http/httputil"
 )
 
 // applyOpenAPIToken 从 openapi 动态获取 oauth2 token
 func (b *Bundle) GetOpenapiOAuth2Token(req apistructs.OpenapiOAuth2TokenGetRequest) (*apistructs.OpenapiOAuth2Token, error) {
```

Comment:
no need this method anymore.
just rename to GetOAuth2Token, and adjust other related structs pls.